book.json

```diff
@@ -9,6 +9,8 @@
     "anchors"
   ],
   "variables": {
-    "org": "Root Systems"
+    "org": "Root Systems",
+    "title": "Enspiral Root Systems NZ",
```

Comment:
Ah I gotcha... I moved them to the top level then inspected it running on the server - the <title> shows up in the <head> but not the <description>... unsure why!